backend/app/rest/api/rest_private.go

```diff
@@ -307,28 +307,42 @@ func (s *private) getEmailCtrl(w http.ResponseWriter, r *http.Request) {
 }
 
 // sendEmailConfirmationCtrl gets address and siteID from query, makes confirmation token and sends it to user.
-// GET /email/subscribe?site=siteID&address=someone@example.com
+// POST /email/subscribe with site and address in json body
```

Comment:
This change is not back-compatible, and I don't see a good reason for this. I'd suggest supporting both existing query params and fallback to the body

Reply:
I've already done that. The difference is that old behaviour is not mentioned in the documentation anymore.

I even left the tests with the old behaviour to verify it still works.

Reply:
my bad, i noticed the removal of address and siteID in diff and have not looked further ;)